epub33/overview/index.html

```diff
@@ -142,15 +188,16 @@ <h3>Reading Order</h3>
 
 					<p>Many types of publication have an obvious reading order, or logical progression through their
 						content. A novel is an example of a highly sequential document — it typically has a beginning,
-						middle and end — but not all publications are so ordered: a cookbook or collection of
+						middle, and end — but not all publications are so ordered: a cookbook or a collection of
 						photographic images might be considered to be more like a database. All documents do, however,
 						have at least one logical ordering of all their top-level content items, whether by date, topic,
-						location, or some other criteria (e.g., a cookbook is typically ordered by type of recipe).</p>
+						location, or some other criteria (e.g., a cookbook is typically ordered by the type of a recipe).</p>
 
 					<p>Each EPUB Publication defines at least one such logical ordering of all its top-level content
-						(the <a href="https://www.w3.org/TR/epub-33/#sec-spine-elem">spine</a> [[EPUB-33]]), as well as
-						a declarative table of contents (the <a>EPUB Navigation Document</a> [[EPUB-33]]). EPUB
-						Publications make these data structures available in a machine-readable way <em>external</em> to
+						(in the <a href="https://www.w3.org/TR/epub-33/#sec-spine-elem">spine</a> [[EPUB-33]] of 
+						the <a href="#sec-package-file">Package Document</a>), as well as
+						through a declarative table of contents (the <a href="#sec-nav-nav-doc">EPUB Navigation Document</a> [[EPUB-33]]). 
```

Comment:
Not sure this works grammatically. If you remove the parentheticals you now have:

> Each EPUB Publication defines at least one such logical ordering of all its top level content, as well as through a declarative table of contents.

Maybe split these two apart so the parentheticals can be dropped:

> Each EPUB Publication defines at least one such logical ordering of all its top-level content in the `<a...>`spine`</a>` of the `<a...>`Package Document`</a>` [[EPUB-33]]. Each also defines a declarative table of contents  in the `<a...>`Package Document`</a>` [[EPUB-33]].